client/src/components/SideDrawer/index.tsx

```diff
@@ -130,21 +131,21 @@ const SideDrawer = (props: any) => {
         </DrawerHeader>
         <Divider />
         <List>
-          {/* TODO update navigate locations */}
           <Box textAlign="center">
             <Button
               variant="contained"
               color="primary"
               style={{ width: "90%" }}
-              onClick={() => navigate("../", { replace: true })}
+              onClick={() => navigate("../create")}
```

Comment:
I would stray away from using relative pathing ESPECIALLY in the sidedrawer since it exists on all pages. instead do something like navigate('create') which will take you to `localhost:3000/create` on development mode and to `roccopolimen.github.io/nextround/create` in production mode. that will be a more robust and uniform way to do nav.